src/Avalonia.Base/Input/HoldingRoutedEventArgs.cs

```diff
@@ -20,6 +20,8 @@ public class HoldingRoutedEventArgs : RoutedEventArgs
         /// </summary>
         public PointerType PointerType { get; }
 
+        public PointerEventArgs? PointerEventArgs { get; }
```

Comment:
It might conflict with https://github.com/AvaloniaUI/Avalonia/pull/11299 later, so I would keep it internal.

Reply:
Done.